index.js

```diff
@@ -184,9 +184,9 @@ module.exports = function (content) {
         urls[format] = path.join(pub, url).replace(/\\/g, '/');
 
         if (generatorConfiguration.dest) {
-          self.emitFile(urls[format], res[format]);
+          this.emitFile(urls[format], res[format]);
```

Comment:
This will break the library (and does break it, by my tests), because this replaces previous context (self) which was webpack's context with global context.
See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/this https://webpack.js.org/api/loaders/#emitfile

I recommend to do this:

```
...
  var self = this;
  var opts = this.options;

  function emitFile(filepath, data) {
    self.emitFile(filepath, data);
  }
...
if (generatorConfiguration.dest) {
    emitFile(urls[format], res[format]);
} else {
    emitFile(url, res[format]);
}
```